modules/oe_content_entity/modules/oe_content_entity_organisation/config/post_updates/00001_create_stakeholder_bundle/core.entity_form_display.oe_organisation.oe_stakeholder.default.yml

```diff
@@ -16,51 +16,45 @@ targetEntityType: oe_organisation
 bundle: oe_stakeholder
 mode: default
 content:
-  created:
-    type: datetime_timestamp
-    weight: 7
-    region: content
-    settings: {  }
-    third_party_settings: {  }
   langcode:
     type: language_select
-    weight: 6
-    region: content
+    weight: 0
+    region: contentproject
```

Comment:
Don't you just hate it when you wanna search for something but PHPStorm doesn't act fast enough and you end up typing something in a random place?

Reply:
k, I'll slow down.